### PR TITLE
WSTEAMA-1426 - Update JumpTo Storybook w/ Fixture Data

### PR DESCRIPTION
**Pull request by pvaliani**
Resolves JIRA [1426](https://jira.dev.bbc.co.uk/browse/WSTEAMA-1426)

Overall changes
======
- Update storybook component w/ fixture data (used Pidgin to visually test even though experiment is Hindi)

### CODE REVIEW

File: src/app/components/JumpTo/index.stories.tsx [hunk 1]
```diff
@@ -17,22 +17,11 @@ export default {
...
-      heading: 'This is a subheading - c',
-    },
-    {
-      heading: 'This is a subheading - d',
-    },
-  ];
+  const jumpToHeadings =
+    pidginArticleFixtureWithJumpToBlock.data.article.content.model.blocks.find(
+      block => block.type === 'jumpTo',
+    )?.model.jumpToHeadings;
```
Comment (pvaliani) [inline src/app/components/JumpTo/index.stories.tsx:24]: Can maybe use helpers to navigate the block in the `Example`? Or fine as is?

**Comment by amoore108** [comment]
Could we follow the pattern in the likes of https://github.com/bbc/simorgh/blob/bde495152e9049e68c0b17417a81962f91c6d91b/src/app/pages/MediaArticlePage/fixtureData.ts, whereby we take just a subset of the ArticlePage data for use as fixture data?

We'd be committing 5k+ lines and only using a small sample of it in the actual Storybook story otherwise.

EDIT: This would co-locate the fixture data to the JumpTo component. So could be extended for the tests as well.

**Comment by amoore108** [comment]
Could the base branch be changed to `WSTEAMA-1421-create-jumpto-skeleton-component` as well? Not sure of Githubs behaviour once `WSTEAMA-1428-update-readme-docs-for-jumpto-component`  gets merged in, and what base branch this PR will automatically switch to.

**Comment by pvaliani** [comment]
> Could we follow the pattern in the likes of https://github.com/bbc/simorgh/blob/bde495152e9049e68c0b17417a81962f91c6d91b/src/app/pages/MediaArticlePage/fixtureData.ts, whereby we take just a subset of the ArticlePage data for use as fixture data?
> 
> We'd be committing 5k+ lines and only using a small sample of it in the actual Storybook story otherwise.
> 
> EDIT: This would co-locate the fixture data to the JumpTo component. So could be extended for the tests as well.

Just updated with it significantly cleaned up. Just so I understand correctly - the example you gave has `fixtureData` co-located with the MAP page but i've co-located next to the component itself, is that correct? Or should it be co-located within `Article`?

The fixture also doesn't currently include the `blockTypes` i.e

```
blockTypes": [
          "headline",
          "text"...
          "jumpTo"]
```

I guess the main thing informing the detail of the fixture is whether or not there are tests/how many/what they are

**Comment by amoore108** [comment]
> Just updated with it significantly cleaned up. Just so I understand correctly - the example you gave has `fixtureData` co-located with the MAP page but i've co-located next to the component itself, is that correct? Or should it be co-located within `Article`?
> 
> The fixture also doesn't currently include the `blockTypes` i.e
> 
> ```
> blockTypes": [
>           "headline",
>           "text"...
>           "jumpTo"]
> ```
> 
> I guess the main thing informing the detail of the fixture is whether or not there are tests/how many/what they are

Yea I'd keep it co-located to the JumpTo component as thats where we're using the data currently. We can extend the `fixtureData.ts` to include more if needed for the tests.

**Review by amoore108 — APPROVED** [review]
Nice, thanks for the changes 👍

**Comment by pvaliani** [comment]
Can I merge this into the main PR now @amoore108 and @louisearchibald?

**Comment by amoore108** [comment]
> Can I merge this into the main PR now @amoore108 and @louisearchibald?

Yea sure 👍

**Pull request merged**